pkgs/servers/nosql/cassandra/3.0.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, jre
+, python
+, makeWrapper
+, gawk
+, bash
+, getopt
+, procps
+}:
+
+let
+
+  version = "3.0.7";
+  sha256 = "0g4nf9zw3by8api9c8np0ixianmwcldcq2mpkqqirj0zlpiii68d";
+
+in
+
+stdenv.mkDerivation rec {
+  name = "cassandra-${version}";
+
+  src = fetchurl {
+    inherit sha256;
+    url = "mirror://apache/cassandra/${version}/apache-${name}-bin.tar.gz";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir $out
+    mv * $out
+
+    for cmd in cassandra nodetool sstableloader sstableupgrade
+      do wrapProgram $out/bin/$cmd \
+        --set JAVA_HOME ${jre} \
+        --prefix PATH : ${bash}/bin \
```

Comment:
A minor stylistic point: I believe all these could be replaced with something like `--prefix PATH : ${makeBinPath [ bash getopt gawk procps ]` (in general, this would also use the correct output implicitly, but it seems unlikely to be an issue here).